javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,8 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var value = [];
+  for (var i = 0; i < values.length; i++) {
+    value.push(values[i]);
+    var noStrings = value.filter(x => typeof x !== 'string')
```

Comment:
If you can tell me exactly how this line works, you can keep it. Otherwise, please do this in a way you fully understand using the tools I have taught you.